README.md

```diff
@@ -1,9 +1,11 @@
 # GDX Death Notification POC
 
 ## API Contracts
+
```

Comment:
Annoyingly my formatter kept seeming to put these in even when I tried to remove them just before committing